docs/builder.rst

```diff
@@ -92,16 +104,57 @@ By default, the manifest builder does *not* perform any validation that the gene
    ethpm.exceptions.ValidationError: Manifest invalid for schema version 2. Reason: '_InvalidPkgName' does not match '^[a-z][-a-z0-9]{0,255}$'
 
 
+To write a manifest to disk
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. code:: python
+
+   to_disk(
+       manifest_root_dir: Optional[Path],
+       manifest_name: Optional[str],
+       prettify: Optional[bool],
+   )
+
+
+Writes the active manifest to disk. Will not overwrite an existing manifest with the same name and root directory.
+
+Defaults
+- Writes manifest to cwd unless a ``Path`` is provided as manifest_root_dir.
+- Writes manifest with a filename of "<version>.json" unless desired manifest name (which must end in ".json") is provided as manifest_name.
+- Writes the minified manifest version to disk unless prettify is set to True.
+
+.. doctest::
+
+   >>> from pathlib import Path
+   >>> import tempfile
+   >>> expected_manifest = """{
+   ...     "package_name": "owned",
+   ...     "manifest_version": "2",
+   ...     "version": "1.0.0"
+   ... }"""
+   >>> p = Path(tempfile.mkdtemp("temp"))
+   >>> build(
+   ...     {},
+   ...     package_name("owned"),
+   ...     manifest_version("2"),
+   ...     version("1.0.0"),
+   ...     to_disk(manifest_root_dir=p, manifest_name="manifest.json", prettify=True),
+   ... )
+   {'package_name': 'owned', 'manifest_version': '2', 'version': '1.0.0'}
+   >>> with open(str(p / "manifest.json")) as f:
+   ...     actual_manifest = f.read()
+   >>> assert actual_manifest == expected_manifest
```

Comment:
For a code example, maybe rather than populating `expected_manifest` you can just load the manifest and then do `print(manifest)`.  The doctest will do the correct comparison with the embedded terminal output.